backend/src/appointment/exceptions/validation.py

```diff
@@ -214,6 +214,9 @@ class RemoteCalendarConnectionError(APIException):
     status_code = 400
 
     def get_msg(self):
+        if self.reason:
+            return self.reason
```

Comment:
This would return the same string as title message and as description for the frontend alert. Imo this could just be left as is, when we're using the corresponding alert component.

Reply:
Thanks, looks like I just needed the frontend change 😄 

Adjusted!